requirements.txt

```diff
@@ -29,4 +29,5 @@ werkzeug>=1.0,<=2.0.3
 multiprocess
 importlib-metadata==4.13.0
 sqlalchemy==1.4.37
-weasyprint==54.3
\ No newline at end of file
+weasyprint==54.3
+notion-client
```

Comment:
Please add the current version number of `notion-client` as well to avoid future break changes from this library.